en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -0,0 +1,212 @@
+# Publishing a sample API to Service Catalog
+
+## What you'll build
+
+In this tutorial we are re-creating the healthcare API we created in the [Sending a Simple Message to a Service](../sending-a-simple-message-to-a-service) tutorial, and publish that to the Service Catalog.
```

Comment:
```suggestion
In this tutorial we are re-creating the healthcare API we created in the [Sending a Simple Message to a Service]({{base_path}}/tutorials/integration-tutorials/sending-a-simple-message-to-a-service) tutorial, and publish that to the Service Catalog.
```

Please use base_path in the links. This is so that the links don't break if the structure changes and it's the strategy we use to consistently manage links in a sustainable manner.

Reply:
Fixed in a0c600c